src/transformers/models/altclip/processing_altclip.py

```diff
@@ -0,0 +1,113 @@
+# coding=utf-8
+# Copyright 2022 WenXiang ZhongzhiCheng LedellWu LiuGuang BoWenZhang The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Image/Text processor class for AltCLIP
+"""
+from ...processing_utils import ProcessorMixin
+from ...tokenization_utils_base import BatchEncoding
+
+
+class AltCLIPProcessor(ProcessorMixin):
+    r"""
+    Constructs a AltCLIP processor which wraps a CLIP feature extractor and a XLM-R tokenizer into a single processor.
+
+    [`AltCLIPProcessor`] offers all the functionalities of [`CLIPFeatureExtractor`] and [`XLMRobertaTokenizerFast`].
+    See the [`~AltCLIPProcessor.__call__`] and [`~AltCLIPProcessor.decode`] for more information.
+
+    Args:
+        feature_extractor ([`CLIPFeatureExtractor`]):
+            The feature extractor is a required input.
+        tokenizer ([`XLMRobertaTokenizerFast`]):
+            The tokenizer is a required input.
+    """
```

Comment:
There should be

here
```
attributes = ["image_processor", "tokenizer"]
```

Reply:
In fact, could you check what is in `CLIPProcessor.__init__`. There are more things missed here. Thank you.

Reply:
It seems transformers is changing Feature Extractor to ImageProcessor. My fork is still using the feature extractor however, so is it ok with no definition of ImageProcessor in my fork?

Reply:
I was trying to test checkpoint file, and I couldn't  instance a processor cause I don't have a ImageProcessor.